### ISSUE TITLE
Provide a default port-forward mapping from the service YAML

### ISSUE THREAD

**Issue by ugumba** [Azure/vscode-kubernetes-tools#659]
The extension should be able to extract the port number(s) or port name(s) from the service YAML, and provide those as a default - e.g. "80" or "http".

**Comment by itowlson** [comment]
Good idea!  I wonder if we can make this work if the service defines multiple ports without blocking the option to type... the VS Code UI can be a bit limiting for that though...
